apollo/src/graphql/auth/auth.ts

```diff
@@ -12,40 +12,32 @@ builder.objectType(ApiKey, {
                 return root.$id();
             },
         }),
+        userId: t.exposeString('userId'),
         apiKey: t.exposeString('apiKey'),
         isArchived: t.exposeBoolean('isArchived'),
         dateCreated: t.exposeString('dateCreated'),
-        user: t.field({
```

Comment:
some of this stuff like user object resolution I'll add back in later, but just want to back out the UUID -> Serial commit for now since I just didn't think that one through